### PR TITLE
Review feedback

### PR DESCRIPTION
**Pull request by jackcmay**
Some friendly feedback, we can discuss in the PR and feel free to ask me any new questions about the program you might have.

### CODE REVIEW

File: program/src/instruction.rs [hunk 1]
```diff
@@ -50,6 +50,8 @@ pub enum Instruction {
     /// 3. `[writable]` The aggregator
     /// 4. `[signer]` The aggregator owner
     AddOracle {
+        // Why add an oracle at a specifc index, could instead add at the next empty submission slot.
```
Comment (czl1378) [inline program/src/instruction.rs:53]: I used to by search for pubkey, then I thought that by index might make the program more robust? 
But by search for pubkey maybe more convenience for developer.
Anyway, I'll consider whether to change this.

File: program/src/state.rs [hunk 2]
```diff
@@ -25,7 +25,12 @@ pub struct Aggregator {
     /// authority
     pub owner: [u8; 32],
     /// submissions
-    pub submissions: [Submission; MAX_ORACLES],
+    // This could be a list of oracle addresses and the other submission data
+    // could be stored in the oracle state, the would remove the need to
+    // repacking the aggregator when submitting.  If the reason is a central
+    // place to look up submission data then could remove the duplicated
+    // `submission` data in the oracle state
```
Comment (czl1378) [inline program/src/state.rs:32]: Yeah, it's a central place to look up submission data, and I'll remove the `submission` in oracle state.

File: program/src/processor.rs [hunk 3]
```diff
@@ -353,7 +359,7 @@ impl Processor {
                 receiver_info.clone(),
                 faucet_owner_info.clone(),
             ],
-            &[&[seed.as_ref()]]
+            &[&[seed.as_ref()]]  // What is this seed for?
         )?;
```
Comment (czl1378) [inline program/src/processor.rs:363]: Can we `invoke_signed` without seed?

Comment (jackcmay) [reply program/src/processor.rs:363]: Yes, you can call `invoke_signed` or just leave the seed slice empty.  Seeds are only required if you are using derived program addresses.

File: program/src/instruction.rs (hunk 1 above)
Comment (jackcmay) [reply program/src/instruction.rs:53]: Index would be easier for developers.  If you do decide to keep index the program should explicitly handle cases where the same oracle is added for different indexes

File: program/src/processor.rs (hunk 3 above)
Comment (czl1378) [reply program/src/processor.rs:363]: Got it, thanks.

File: program/src/instruction.rs (hunk 1 above)
Comment (czl1378) [reply program/src/instruction.rs:53]: Sure.

**Comment by defactojob** [comment]
outdated. closing.

**Pull request closed without merging**